### PR TITLE
Script to automate creation of terraform resources on multiple AWS accounts

### PR DESCRIPTION
**Pull request by ziemowit141**
Terraform module changed in order to allow working with multiple AWS users within single account (resource names are distinct per workspace)

TODO:
- add README

### CODE REVIEW

File: cloud_AWS/terraform/module/examples/multiple_aws_accounts_single_vpc_setup/configure_aws_accounts.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+        return_code, stdout, stderr = t.create_workspace(credentials_set["workspace"])
+        print(return_code, stdout, stderr)
+
+        if f'Workspace "{credentials_set["workspace"]}" already exists' in stderr:
+            return_code, stdout, stderr = t.set_workspace(credentials_set["workspace"])
+            print(return_code, stdout, stderr)
+
+        t.plan()
+        return_code, stdout, stderr = t.apply(var={'vpc_id': "vpc-0eeb173de06765d0a"}, skip_plan=True)
```
[the remark this answers is not in the document]
Comment (mmac-m3a) [reply cloud_AWS/terraform/module/examples/multiple_aws_accounts_single_vpc_setup/configure_aws_accounts.py:70]: I envision 3 scenarios  for export of flow logs to Kentik for customers with large number of accounts:
1) all VPCs in the account
2) VPC matching some criteria (e.g. specific tag, or combination of tags)
3) explicit list VPC ids (ARNs)

The 3rd alternative is though the least likely, especially for customers with large number accounts and/or VPCs.

[the remark this answers is not in the document]
Comment (mateuszmidor) [reply cloud_AWS/terraform/module/examples/multiple_aws_accounts_single_vpc_setup/configure_aws_accounts.py:70]: OK, will use all available VPCs to begin with, we can later add some sort of vpc selection

**Comment by mmac-m3a** [comment]
This PR can be discarded, right?

**Comment by mateuszmidor** [comment]
> This PR can be discarded, right?

Yes, correct.

**Comment by mmac-m3a** [comment]
Replaced by https://github.com/kentik/config-snippets-cloud/pull/41

**Pull request closed without merging**